plugins/accessibility/index.js

```diff
@@ -0,0 +1,88 @@
+var q = require('q'),
+    fs = require('fs'),
+    path = require('path'),
+    _ = require('lodash');
+
+var AUDIT_FILE = path.join(__dirname, '../../node_modules/accessibility-developer-tools/dist/js/axs_testing.js');
+
+/**
+ * You enable this plugin in your config file:
+ *
+ *    // Chrome Accessibility Developer Tools:
+ *    exports.config = {
+ *      ...
+ *      plugins: {
+ *        ...
+ *        accessibility: {
+ *          developerTools: true,
+ *        }
+ *      }
+ *    }
+ */
+
+/**
+ * Checks the information returned by the accessibility audit and
+ * displays passed/failed results as console output
+ *
+ * @param {Object} config The configuration file for the accessibility plugin
+ * @return {q.Promise} A promise which resolves to the results of any passed or
+ *    failed tests
+ * @public
+ */
+function teardown(config) {
+
+  var data = fs.readFileSync(AUDIT_FILE, 'utf-8');
+  data = data + ' return axs.Audit.run();';
+
+  var testOut = {failedCount: 0, specResults: []},
+      elementPromises = [];
+
+  return browser.executeScript_(data, 'axs rules').then(function(results) {
```

Comment:
let's make the description `a11y developer tool rules` since I won't rememberwhat axs means when debugging